models/_sector/nft/trades/solana/nft_solana_trades.sql

```diff
@@ -0,0 +1,83 @@
+{{
+    config(
+        schema = 'nft_solana'
+        , tags = ['dunesql']
+        , alias = alias('trades')
+        , materialized = 'view'
+        ,post_hook='{{ expose_spells(\'["solana"]\',
+                                    "sector",
+                                    "nft",
+                                    \'["ilemi"]\') }}'
+    )
+}}
+
+-- ,file_format = 'delta'
+-- ,incremental_strategy = 'merge'
+-- ,partition_by = ['project','block_month']
+-- ,unique_key = ['project','trade_category','outer_instruction_index','inner_instruction_index','account_metadata','tx_id']
```

Comment:
can we remove these to avoid confusion?

if you ever materialize in the future, should be able to copy/paste config options from any underlying project spell, as the goal is to have same unique key throughout